src/components/CornerCircularLabel/CornerCircularLabel.scss

```diff
@@ -0,0 +1,36 @@
+.corner-circular-label {
+  position: absolute;
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  border-radius: 50%;
+  width: 3.5vmin;
+  height: 3.5vmin;
+  line-height: 3.5vmin;
+  text-align: center;
+  text-decoration: none;
+  background-color: #c300c3;
+  font-size: 1.25vh;
+  font-weight: bold;
+  color: white;
+
+  &.top-right {
+    right: -1em;
```

Comment:
When using screen-relative units like `vh`, `vmin` etc., it's best to keep using them everywhere (`em` is not screen-relative and depends on the unit size of the parent container and as such should be avoided).